packages/core/__tests__/typescript/effects.ts

```diff
@@ -754,6 +754,14 @@ function* testTakeEvery(): SagaIterator {
   yield takeEvery('my-action', helperWorker7, 'a', 'b', 'c', 'd', 'e', 'f')
   yield takeEvery('my-action', helperWorker7, 'a', 'b', 'c', 'd', 'e', 'f', 'g')
 
+  const helperWorker8 = (a: 'a', b: 'b', c: 'c', d: 'd', e: 'e', f: 'f', g: 'g') => {}
```

Comment:
I've copied `helperWorker7`, but without `action` param to test the new type behavior. Do you want more tests ?